### PR TITLE
Adding Example for Prometheus Pushgateway

### PR DESCRIPTION
**Pull request by ChamilaLiyanage**
Added a Prometheus Pushgateway deployment and the scrape configuration required for that.

### CODE REVIEW

**Comment by ChamilaLiyanage** [comment]
Review request.

File: monitoring/pushgateway/pushgateway.yaml [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+     matchLabels:
+       app: pushgateway
+   template:
+      metadata:
+         labels:
+           app: pushgateway
+      spec:
+        containers:
+         - name: pushgateway
+           image: prom/pushgateway
```
Comment (Thakurvaibhav) [inline monitoring/pushgateway/pushgateway.yaml]: Please pin this to a version.

**Comment by Thakurvaibhav** [comment]
> Added a Prometheus Pushgateway deployment and the scrape configuration required for that.

Can you please update the README.md to reflect the purpose of adding pushgateway. A couple of line should be enough.

**Comment by ChamilaLiyanage** [comment]
> > Added a Prometheus Pushgateway deployment and the scrape configuration required for that.
> 
> Can you please update the README.md to reflect the purpose of adding pushgateway. A couple of line should be enough.

Added description to the readme file.

File: monitoring/pushgateway/pushgateway.yaml (hunk 1 above)
Comment (ChamilaLiyanage) [reply monitoring/pushgateway/pushgateway.yaml]: Pinned to version v1.0.1

**Comment by Thakurvaibhav** [comment]
@ChamilaLiyanage  Approved. Thanks!

**Pull request merged**